src/SourceBuild/content/repo-projects/Directory.Build.targets

```diff
@@ -66,6 +66,7 @@
       <ExtraSourcesNuGetSourceName>ExtraSources</ExtraSourcesNuGetSourceName>
       <SourceBuildSources>$(SourceBuiltNuGetSourceName)</SourceBuildSources>
       <SourceBuildSources Condition="'$(ExtraRestoreSourcePath)' != ''">$(SourceBuildSources);$(ExtraSourcesNuGetSourceName)</SourceBuildSources>
+      <NetSdkSupportingFeedName>net-sdk-supporting-feed</NetSdkSupportingFeedName>
```

Comment:
What's the reason for defining the feed url in the props versus centralizing the two values?  From a maintenance perspective having them next to each other would be nice.

Reply:
I thought it followed the pattern of the other feeds more closely. The feeds are defined in the props files (directories mostly), but the names are here.

Would you prefer to have the values in the targets files?

Reply:
I see those as different in that they are local paths in which the path properties are shared amongst various other places (e.g. the logic which populates the local cache).  This one is not shared so that reasoning doesn't apply.

Not a big deal but that is my perspective.

Reply:
Moved. let me know what you think.